src/alpaca_eval/models_configs/openbuddy-llama-65b-v8/configs.yaml

```diff
@@ -0,0 +1,13 @@
+openbuddy-llama-65b-v8:

```

Comment:
I don't see the evaluations for this model, can you remove this config from the PR or evaluate the model?